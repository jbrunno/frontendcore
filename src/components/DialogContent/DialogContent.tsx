import React from 'react';
import {
  DialogContent as MuiDialogContent,
  DialogContentProps as MuiDialogContentProps,
} from '@mui/material';

export type DialogContentProps = MuiDialogContentProps;

export function DialogContent(props: DialogContentProps) {
  return <MuiDialogContent {...props} />;
}

export default DialogContent;
