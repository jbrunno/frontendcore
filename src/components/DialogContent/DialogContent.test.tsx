import React from 'react';
import { render } from '@test-utils/component-testing';
import { DialogContent } from './DialogContent';

test('renders the DialogContent', () => {
  const { container } = render(<DialogContent />);
  expect(container).toMatchSnapshot();
});
