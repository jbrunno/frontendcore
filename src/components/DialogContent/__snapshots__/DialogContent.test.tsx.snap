// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the DialogContent 1`] = `
.emotion-0 {
  -webkit-flex: 1 1 auto;
  -ms-flex: 1 1 auto;
  flex: 1 1 auto;
  -webkit-overflow-scrolling: touch;
  overflow-y: auto;
  padding: 20px 24px;
}

.MuiDialogTitle-root+.emotion-0 {
  padding-top: 0;
}

<div>
  <div
    class="MuiDialogContent-root emotion-0"
  />
</div>
`;
